pkg/apis/serving/v1/revision_types.go

```diff
@@ -88,6 +88,18 @@ type RevisionSpec struct {
 	// (send network traffic). If unspecified, a system default will be provided.
 	// +optional
 	TimeoutSeconds *int64 `json:"timeoutSeconds,omitempty"`
```

Comment:
We should update the timeout description

https://github.com/knative/specs/pull/100/files